applications/CompressiblePotentialFlowApplication/custom_utilities/potential_flow_utilities.cpp

```diff
@@ -0,0 +1,56 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Inigo Lopez
+
+#include "custom_utilities/potential_flow_utilities.h"
+
+namespace Kratos
+{
+namespace PotentialFlowUtilities
+{
+template <int Dim, int NumNodes>
+array_1d<double, NumNodes> GetPotentialOnNormalElement(const Element& rElement)
+{
+    const int kutta = rElement.GetValue(KUTTA);
+    array_1d<double, NumNodes> potentials;
+
+    const Element::GeometryType& Geometry = rElement.GetGeometry();
+
+    if (kutta == 0)
+    {
+        for (unsigned int i = 0; i < NumNodes; i++)
+        {
+            potentials[i] = Geometry[i].FastGetSolutionStepValue(VELOCITY_POTENTIAL);
+        }
+    }
+    else
+    {
+        for (unsigned int i = 0; i < NumNodes; i++)
```

Comment:
the brackets should be on the same line

Reply:
In https://github.com/KratosMultiphysics/Kratos/wiki/How-to-configure-clang%E2%80%90format#clang-format-file it is stated that BreakBeforeBraces: Allman should be used. However I think it should be BreakBeforeBraces: Stroustrup as you say

Reply:
hm interesting
@loumalouomega what do you say?

Reply:
For now I changed it in the wiki, but let me know if I should change it back. @philbucher and @loumalouomega  what settings do you have for the BreakBeforeBraces: in your clang format file?